runners/wikiConsole/templates/installDevonfwIde.asciidoc

```diff
@@ -11,6 +11,19 @@ Create a new directory in the location where you want to install the devonfw IDE
 
 Download
 ~~~~~~~~~
+
+<% if(version) {%>
+
+Now you have to download version <%= version; %> of the devonfw IDE. You can download it from https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.devonfw.tools.ide&a=devonfw-ide-scripts&v=<%= version; %>&p=tar.gz[here].
+
+In the Terminal execute `Invoke-WebRequest -OutFile devonfw-ide-scripts.tar.gz 'https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.devonfw.tools.ide&a=devonfw-ide-scripts&v=<%= version; %>&p=tar.gz'` if you are using Windows.
+
+On Linux use `wget -c 'https://repository.sonatype.org/service/local/artifact/maven/redirect?r=central-proxy&g=com.devonfw.tools.ide&a=devonfw-ide-scripts&v=   <%= version; %>&p=tar.gz' -O devonfw-ide-scripts.tar.gz -`.
+
+After that you need to extract the downloaded file. To do this, run `tar -xvzf devonfw-ide-scripts.tar.gz`. Or you can simply use a tool (e.g. 7-Zip) for this.
```

Comment:
This line in now redundant in the template. You can just move it below the if part